.github/workflows/snakemake-ci.yml

```diff
@@ -0,0 +1,65 @@
+name: Snakemake CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
These should be `main` instead of `master` given the default branch name, so this workflow doesn't actually run.